### PR TITLE
Updating nu get packages

### PR DESCRIPTION
**Pull request by HenrikRxn**
First part of long PR. Getting Domain.UnitTests under NuGet

### CODE REVIEW

**Comment by HenrikRxn** [comment]
@parthopdas Please review

The appveyor build runner only picks up 120 instead of 149 tests. I wrote a question on Gitter regarding this, but appveyor is very poor when it comes to investigating the difference in executed tests between runs

File: .gitignore [hunk 1]
```diff
@@ -159,7 +159,7 @@ PublishScripts/
 ## The packages folder can be ignored because of Package Restore
```
Comment (parthopdas) [inline .gitignore]: Can you please uncomment lines 157 to 167. And delete lines 168 to 170. (I had to make these changes as we didnt have paket when I copied github's standard vs ignore file)

**Comment by parthopdas** [comment]
Looks great overall. And sharp eye catching the discrepancy in the unit tests.

Lets fix the testhost.core.unittests issue as well so all unit test are running.

File: .gitignore (hunk 1 above)
Comment (HenrikRxn) [inline .gitignore]: We still do not have Paket :-)

It seems to me that the uncomment of 157-167 requires package restore and that we should remove all content of the packages folder?

IMHO this is not related to the issue of this PR, so I'll do these changes when I do the Paket stuff

**Pull request merged**